mmv1/api/resource.rb

```diff
@@ -396,7 +396,11 @@ def all_nested_properties(props)
     def convert_go_file(file)
       dir, base = File.split(file)
       base.slice! '.erb'
-      "#{dir}/go/#{base}.tmpl"
+      if dir.end_with?('terraform')
+        "#{dir}/#{base}.go.tmpl"
```

Comment:
One example is `templates/terraform/update_mask.erb` -> `templates/terraform/update_mask.go.tmpl`